cmd/neofs-node/config.go

```diff
@@ -88,27 +80,45 @@ const notificationHandlerPoolSize = 10
 // values. It should not store any application helpers structs (pointers to shared
 // structs).
 // It must not be used concurrently.
+//
+// REFACTOR IS NOT FINISHED.
 type applicationConfiguration struct {
 	// _read indicated whether a config
 	// has already been read
 	_read bool
 
-	LoggerCfg struct {
+	Logger struct {
 		level string
 	}
 
-	EngineCfg struct {
+	Engine struct {
 		errorThreshold uint32
 		shardPoolSize  uint32
 		shards         []storage.ShardCfg
 	}
 
-	PolicerCfg struct {
+	Policer struct {
```

Comment:
Questionable to me. It's still some config in most cases. Processed, ready to go, but not some component.

Reply:
we _literally_ have both `MorphCfg` and `cfgMorph` in the code. basic usage of _this_ struct is: `c.applicationConfiguration.Morph` so that is why I think _this_ struct can miss `Cfg` suffix (`applicationConfiguration` is the struct that has exact values from a `yml` file and the other sources by my design)

Reply:
struct renamings are welcome